backend/routes/questionnaires/questionnaires.js

```diff
@@ -13,7 +13,7 @@ router.route('/').get((req, res) => {
 
 router.route('/:title.:language').get((req, res) => {
     Questionnaires.findOne({
-        title: req.params.title,
+        title: decodeURIComponent(req.params.title),
```

Comment:
why is this needed here?

Reply:
in case the title has any special characters or spaces